### PR TITLE
Section/icons

### PR DESCRIPTION
**Pull request by whiteblackswan**
This is link to netlify.

https://tranquil-sorbet-a8923c.netlify.app/

### CODE REVIEW

**Comment by Dmitruk89** [comment]
image
Please reduce the height of sections on smaller screens, mainly to avoid the green section appeared in bottom of slider.

**Comment by Dmitruk89** [comment]
image
on the breakpoint about 910px, the height of the slides start jumping causing overlap with the elements on the buttom

File: index.html [hunk 1]
```diff
@@ -51,15 +51,68 @@ <h4 class="section__subtitle">We are a professional digital agency that
...
 					<span>Get Started</span>
-					<img src="icons/ArrowRight.png"alt="Arrow">
+					<img src="icons/ArrowRight.png" alt="Arrow">
 				</button>
 
 			</div>
 		</section>
+
+		<div class="slider">
+			<div class="container">
```
Comment (Dmitruk89) [inline index.html:64]: this is slider container, so please change the name to "slider__container"

File: index.html [hunk 2]
```diff
@@ -51,15 +51,68 @@ <h4 class="section__subtitle">We are a professional digital agency that
...
+				<div class="slide-container">
```
Comment (Dmitruk89) [inline index.html:65]: according to the BEM, name this class as slide__container

File: index.html [hunk 3]
```diff
@@ -51,15 +51,68 @@ <h4 class="section__subtitle">We are a professional digital agency that
...
+						<img src="img/list.png" alt="">
+					</div>
+					<div class="slide fade">
+						<img src="img/Untitled.png" alt="">
+					</div>
+
+					<a href="#" class="prev arrow" title="Previous"><img src="icons/left-arrow-mini.png" alt="Arrow"></a>
+					<a href="#" class="next arrow" title="Next"><img src="icons/arrow-right-mini.png" alt="Arrow"></a>
+				</div>
+				<div class="dots-container">
```
Comment (Dmitruk89) [inline index.html:82]: the same here: "dots__container" or better "dot-list"

File: index.html [hunk 4]
```diff
@@ -51,15 +51,68 @@ <h4 class="section__subtitle">We are a professional digital agency that
...
+					<span class="dot"></span>
+					<span class="dot"></span>
+					<span class="dot"></span>
+					<span class="dot"></span>
+				</div>
+			</div>
+		</div>
+
+		<section class="icons">
+			<div class="container">
```
Comment (Dmitruk89) [inline index.html:92]: if this container common for all the section, you may call it "section__container"

File: index.html [hunk 5]
```diff
@@ -51,15 +51,68 @@ <h4 class="section__subtitle">We are a professional digital agency that
...
+				<div class="dots-container">
+					<span class="dot"></span>
+					<span class="dot"></span>
+					<span class="dot"></span>
+					<span class="dot"></span>
+				</div>
+			</div>
+		</div>
+
+		<section class="icons">
```
Comment (Dmitruk89) [inline index.html:91]: you may call this block "statistic" then rename the classes "flex-container" and "flex-block" according to the BEM

File: js/script.js [hunk 6]
```diff
@@ -7,3 +7,43 @@ menuButton.addEventListener('click', () => {
 	navLinks.classList.toggle('active');
 	menuButton.classList.toggle('active');
 });
+
+var currentSlide = 0;
```
Comment (Dmitruk89) [inline js/script.js:11]: use "let" here

File: sass/_slider.sass [hunk 7]
```diff
@@ -0,0 +1,74 @@
+.slider
+	position: absolute
+	left: 0
+	right: 0
+	margin-left: auto
+	margin-right: auto
```
Comment (Dmitruk89) [inline sass/_slider.sass:6]: this two properties gives no effect, you may clear them

File: sass/_slider.sass [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+	margin-top: -250px
```
Comment (Dmitruk89) [inline sass/_slider.sass:7]: giving negative margin is bad practice. Try find another solution (e.g. using "top" with negative values is ok)

File: sass/_slider.sass [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+	position: relative
+.slide-container .slide
+	width: 100%
+	max-height: 630px
+	overflow: hidden
+.slide-container .slide.fade
+	animation: fade 0.5s cubic-bezier(0.55, 0.085, 0.68, 0.53) both
+.slide-container .slide img
+	width: 100%
+
```
Comment (Dmitruk89) [inline sass/_slider.sass:22]: remove this empty line

File: sass/_slider.sass [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+	
+	position: relative
+
+.slide-container .prev,
+.slide-container .next
+	cursor: pointer
+	position: absolute
+	top: 50%
+	width: auto
+	margin-top: -22px
```
Comment (Dmitruk89) [inline sass/_slider.sass:31]: the same negative margin, please replace it with something else

File: sass/_slider.sass [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+	font-weight: bold
+	font-size: 20px
+	transition: all 0.6s ease
+	border-radius: 0 3px 3px 0
+	user-select: none
+
+.slide-container .prev:hover,
+.slide-container .next:hover
+	background-color: rgba(0, 0, 0, 0.2)
+	color: white
```
Comment (Dmitruk89) [inline sass/_slider.sass:43]: I guess you use constants for colors

File: sass/_slider.sass [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+	display: flex
+	justify-content: center
+	align-items: center
+	padding: 10px
+.dots-container .dot
+	cursor: pointer
+	margin: 5px
+	width: 20px
+	height: 20px
+	color: #333
```
Comment (Dmitruk89) [inline sass/_slider.sass:59]: please use constants for colors

**Review by Dmitruk89 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (whiteblackswan) [reply index.html:64]: You're right, that's make sense.

Comment (whiteblackswan) [reply index.html:64]: I checked this, and on line 63, I have a container that holds the entire width.

File: index.html (hunk 2 above)
Comment (whiteblackswan) [reply index.html:65]: This makes sense, I fixed it.

File: index.html (hunk 4 above)
Comment (whiteblackswan) [reply index.html:92]: I fixed it.

File: index.html (hunk 5 above)
Comment (whiteblackswan) [reply index.html:91]: I fixed it

File: sass/_slider.sass (hunk 9 above)
Comment (whiteblackswan) [reply sass/_slider.sass:22]: I fixed it.

File: sass/_slider.sass (hunk 11 above)
Comment (whiteblackswan) [reply sass/_slider.sass:43]: I fixed it.

File: sass/_slider.sass (hunk 12 above)
Comment (whiteblackswan) [reply sass/_slider.sass:59]: I fixed it.

File: sass/_slider.sass (hunk 8 above)
Comment (whiteblackswan) [reply sass/_slider.sass]: I fixed it

File: sass/_slider.sass (hunk 10 above)
Comment (whiteblackswan) [reply sass/_slider.sass:29]: I fixed it.

File: index.html [hunk 13]
```diff
@@ -51,15 +51,68 @@ <h4 class="section__subtitle">We are a professional digital agency that
...
+			</div>
+		</div>
+
+		<section class="statistic">
+			<div class="container">
+				<div class="statistic__container">
+					<div class="statistic__block">
+						<img src="icons/1.2K+.png" alt="Arrow">
+						<p>Happy Clients</p>
+					</div>
```
Comment (Dmitruk89) [inline index.html:97]: it is more proper to use name "statistic__item" as statistic block - it is section "statistic". other mentions of block causes some confusions.

Comment (whiteblackswan) [reply index.html:97]: You are right, I didn't notice.

File: js/script.js (hunk 6 above)
Comment (whiteblackswan) [reply js/script.js:11]: I fixed it

**Review by Dmitruk89 — APPROVED** [review]

**Pull request merged**